### PR TITLE
Added pushover plugin and option to mute 'soft' advice

### PR DESCRIPTION
**Pull request by 0b01**
#398

### CODE REVIEW

**Comment by 0b01** [comment]
@askmike this PR is ready to merge.

File: plugins/adviceLogger.js [hunk 1]
```diff
@@ -16,9 +19,10 @@ Actor.prototype.processCandle = function(candle, done) {
 };
 
 Actor.prototype.processAdvice = function(advice) {
+  if (adviceLoggerConfig.muteSoft && advice.recommendation == 'soft') return;
   console.log()
   log.info('We have new trading advice!');
-  log.info('\t Position:', advice.recommendation);
+  log.info('\t Position:', advice.recommandation);
```
Comment (FFMG) [inline plugins/adviceLogger.js]: It is ``advice.recommendation`` not ``advice.recommandation``, (wrong spelling)

**Comment by askmike** [comment]
that looks great! Unfortunately I am not in a position to test this out, could you therefor submit this PR against the askmike:develop branch? I'll test it out as soon as I can, thanks already!

**Comment by 0b01** [comment]
done

**Pull request closed without merging**